src/test/regress/sql/multi_extension.sql

```diff
@@ -476,9 +484,17 @@ SELECT * FROM multi_extension.print_extension_changes();
 ALTER EXTENSION citus UPDATE TO '11.0-3';
 SELECT * FROM multi_extension.print_extension_changes();
 
--- Test downgrade to 11.0-3 from 11.1-1
-ALTER EXTENSION citus UPDATE TO '11.1-1';
+-- Snapshot of state at 11.0-4
+ALTER EXTENSION citus UPDATE TO '11.0-4';
+SELECT * FROM multi_extension.print_extension_changes();
+
+-- Test downgrade script (result should be empty)
 ALTER EXTENSION citus UPDATE TO '11.0-3';
+ALTER EXTENSION citus UPDATE TO '11.0-4';
+
+-- Test downgrade to 11.0-4 from 11.1-1
+ALTER EXTENSION citus UPDATE TO '11.1-1';
+ALTER EXTENSION citus UPDATE TO '11.0-4';
```

Comment:
The order of tests is not really correct here.

Assume we have versions X, Y and Z in increasing order. We usually want the following in our tests:
- Upgrade from X to Y
- print extension schema changes
- downgrade to X, immediately followed by an upgrade to Y.
- verify that there is no schema change after downgrade+upgrade
- Upgrade from Y to Z
- ...

For example, the following is the correct order of operations:
```sql
-- Snapshot of state at 11.0-2
ALTER EXTENSION citus UPDATE TO '11.0-2';
SELECT * FROM multi_extension.print_extension_changes();
-- Test downgrade to 11.0-2 from 11.0-3
ALTER EXTENSION citus UPDATE TO '11.0-3';
ALTER EXTENSION citus UPDATE TO '11.0-2';
```

However the following is not:
```sql
-- Snapshot of state at 11.0-4
ALTER EXTENSION citus UPDATE TO '11.0-4';
SELECT * FROM multi_extension.print_extension_changes();

-- Test downgrade script (result should be empty)
ALTER EXTENSION citus UPDATE TO '11.0-3';
ALTER EXTENSION citus UPDATE TO '11.0-4'; -- we should print extension changes to observe the absence of schema changes right after this

-- Test downgrade to 11.0-4 from 11.1-1
ALTER EXTENSION citus UPDATE TO '11.1-1'; -- we should print the snapshot at 11.1-1 first, and test the downgrades afterwards.
ALTER EXTENSION citus UPDATE TO '11.0-4';
```

I believe this PR looks good except for the tests here.